eng/tools/BaselineGenerator/BaselineGenerator.csproj

```diff
@@ -3,12 +3,11 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <StartArguments>-o "$(MSBuildThisFileDirectory)../../Baseline.Designer.props"</StartArguments>
```

Comment:
Why is this removed?

Reply:
1. It conflicts with `--update`
2. It's not useful because `$(StartWorkingDirectory)` sets the current directory correctly and the tool defaults to finding `Baseline.Designer.props` in its current directory